### DISCUSSION TITLE
Customization of organisation data for GSF learning and exam results

### DISCUSSION THREAD

**Discussion by PindyBhullar** [Green-Software-Foundation/discussions#18] [Community Working Group]
I want to integrate the GSF training into our internal learning platform. 

How do we get access of our employees that have undertaken the GSF learning and then the results of the exam?
Do you have an API that can be used for regular updates/results?

Thanks.

**Comment by seanmcilroy29** [comment]
Once the course is completed, Orgs can include a link that enables members to confirm their course completion by submitting their details.
![image](https://github.com/Green-Software-Foundation/community-wg/assets/8318213/78cd9e64-38a7-4116-b272-54c566683455)

**Comment by PindyBhullar** [comment]
Thank you Sean.
